images/bundle/importer.go

```diff
@@ -0,0 +1,249 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+// Package bundle provides a Docker and OCI compatible importer
+package bundle
```

Comment:
nit: I would expect "bundle" to mean OCI runtime bundle (`config.json` + `rootfs`)

Maybe we can call this package just "oci"

Reply:
Do we use any other term to represent this tar archive? I don't think `oci` is the right package name here, it is a subset of the OCI functionality and now includes functionality which is not OCI.